app/code/community/Aoe/JsCssTstamp/Block/Head.php

```diff
@@ -340,11 +340,15 @@ protected function &_prepareStaticAndSkinElements($format, array $staticItems, a
     protected function _reorderItems(array $items)
     {
         if (isset($items[''])) {
-            $defaultItem = $items[''];
+            $defaultItem = [];
+            array_push($defaultItem, $items['']);
             unset($items['']);
             ksort($items);
-            $items = array_values($items);
-            array_push($items, $defaultItem);
+            $items = array_pop($items);
+            if(!empty($items)) {
```

Comment:
Actually, looking at this, it would break multiple array keys, as only 1 will be popped off and used, dropping the rest, which will also cause an issue